### PR TITLE
refact: Package and code splitting

### PR DESCRIPTION
**Pull request by marknefedov**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -25,19 +25,22 @@ jobs:
       - name: Install swaggo
         run: go install github.com/swaggo/swag/cmd/swag@v1.8.12
 
-      - name: Generate Go ebpf objects
-        run: go generate -v ./cmd/eupf
+      - name: Generate project dependencies
```
Comment (pirog-spb) [inline .github/workflows/test.yml:28]: Questionably. Whick kind of deps are updated here? Compilation of ebpf program doesn't look like deps IMHO

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -25,19 +25,22 @@ jobs:
...
+        run: go vet ./cmd/...
 
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v3
         with:
           working-directory: ./cmd/eupf
 
-      - name: Test
-        run: sudo go test -v ./cmd/eupf
+      - name: Test core
```
Comment (pirog-spb) [inline .github/workflows/test.yml:42]: Run tests?

File: cmd/eupf/ebpf/ebpf_pdr.go
Comment (pirog-spb) [inline cmd/eupf/ebpf/ebpf_pdr.go:1]: `ebpf/ebpf_pdr.go` -> `ebpf/pdr.go`

Comment (pirog-spb) [reply cmd/eupf/ebpf/ebpf_pdr.go:1]: Apply the same for all `ebpf/ebpf_*.go` files

File: cmd/eupf/main.go [hunk 3]
```diff
@@ -1,6 +1,8 @@
 package main
 
 import (
+	"github.com/edgecomllc/eupf/cmd/eupf/core"
```
Comment (pirog-spb) [inline cmd/eupf/main.go:4]: `eupf` apears twice in path `github.com/edgecomllc/eupf/cmd/eupf/core`. Looks redundant

File: .github/workflows/test.yml (hunk 1 above)
Comment (marknefedov) [reply .github/workflows/test.yml:28]: The spread syntax `./cmd/...` runs go generate for all folders in cmd, that include api, where swagger files are generated.

File: cmd/eupf/main.go (hunk 3 above)
Comment (marknefedov) [reply cmd/eupf/main.go:4]: That how project was laid out from the start, than we need to move everything from cmd/eupf folder to cmd.

File: .github/workflows/test.yml (hunk 1 above)
Comment (pirog-spb) [reply .github/workflows/test.yml:28]: I'm talking about `name`

File: cmd/eupf/ebpf/ebpf_objects.go [hunk 4]
```diff
@@ -12,19 +12,19 @@ import (
...
 )
 
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target bpf ip_entrypoint 	xdp/n3n6_entrypoint.c -- -I. -O2 -Wall -g
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target bpf zero_entrypoint 	xdp/zero_entrypoint.c -- -I. -O2 -Wall
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target bpf n3_entrypoint 	xdp/n3_entrypoint.c -- -I. -O2 -Wall
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target bpf n6_entrypoint 	xdp/n6_entrypoint.c -- -I. -O2 -Wall
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target bpf qer_program 		xdp/qer_program.c -- -I. -O2 -Wall
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target bpf far_program 		xdp/far_program.c -- -I. -O2 -Wall
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target bpf upf_xdp 			xdp/upf_program.c -- -I. -O2 -Wall
+//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target bpf Ip_entrypoint 	xdp/n3n6_entrypoint.c -- -I. -O2 -Wall -g
```
[the remark this answers is not in the document]
Comment (marknefedov) [reply cmd/eupf/ebpf/ebpf_objects.go]: I'll proceed with camel case than, Go uses capitalization for marking public/private variables.

File: cmd/eupf/main.go (hunk 3 above)
Comment (pirog-spb) [reply cmd/eupf/main.go:4]: Why not to move right now?

File: .github/workflows/test.yml (hunk 1 above)
Comment (marknefedov) [reply .github/workflows/test.yml:28]: Well then, what name do you suggest?

Comment (pirog-spb) [reply .github/workflows/test.yml:28]: Previous name looked good. May be `Compile ebpf and generate go wrappers`?

**Review by pirog-spb — APPROVED** [review]

**Pull request merged**